projects/storefrontapp/server.ts

```diff
@@ -10,7 +10,7 @@ import { AppServerModule } from './src/main.server';
 // And we need to use esModuleInterop option in ssr dev mode, because i18next enforce usage of this option for cjs module.
 const express = require('express');
 
-const ngExpressEngine = NgExpressEngineDecorator.get(engine);
+const ngExpressEngine = NgExpressEngineDecorator.get(engine, null);
```

Comment:
I'm in favor of keeping our SSR optimization engine enabled (as a part of the testing).
We can tweak the options thought.

Reply:
I reverted this. Also, increasing the timeout by 2 seconds did the trick. Let me know if that's ok or we should do something else